pkgs/dartpad_ui/lib/local_storage.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'local_storage/stub.dart'
```

Comment:
Do we need this library? Why can't we use the [localStorage API from package:web](https://pub.dev/documentation/web/latest/web/StorageAccessHandle/localStorage.html)?

Reply:
Habits of writing cross-platform code, but of course this is a web-only project. Removed the conditional import and replaced it with the real web implementation. 

I was using `web.window.localStorage` though, instead of the API you linked, `web.document.requestStorageAccess()`. Given that DartPad is often embedded into the docs and codelabs, do I need to switch to that?

Reply:
No, window.localStorage (from package:web) is the correct API, I sent the wrong link.